karspexet/venue/management/commands/build_seats.py

```diff
@@ -0,0 +1,34 @@
+from django.core.management.base import BaseCommand, CommandError
+from karspexet.venue.models import Venue, Seat, SeatingGroup
+
+import csv
+
+class Command(BaseCommand):
+    def add_arguments(self, parser):
+        parser.add_argument("venue_id", type=int)
```

Comment:
This should probably be `venue-id` to stay unixy.

Reply:
Good point.